autoload/ale/fixers/hackfmt.vim

```diff
@@ -0,0 +1,17 @@
+" Author: Sam Howie <samhowie@gmail.com>
+" Description: Integration of hackfmt with ALE.
+
+call ale#Set('php_hackfmt_executable', 'hackfmt')
+call ale#Set('php_hackfmt_options', '-i')
```

Comment:
What is the meaning of `-i`? If it's essential to running the program, you might leave that out of configuration.

Reply:
`-i` id the in-place option. It tells hackfmt to format the file in place.

It is essential, so I will move it out of the configuration.